app/uk/gov/hmrc/singlecustomeraccountwrapperdata/controllers/ScaWrapperController.scala

```diff
@@ -17,60 +17,32 @@
 package uk.gov.hmrc.singlecustomeraccountwrapperdata.controllers
 
 import play.api.Logging
-import play.api.http.HeaderNames
 import play.api.i18n.{I18nSupport, Lang}
 import play.api.libs.json.Json
 import play.api.mvc.{Action, AnyContent, ControllerComponents}
 import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
-import uk.gov.hmrc.singlecustomeraccountwrapperdata.config.{AppConfig, UrBanner, UrBannersConfig, WebchatConfig, WrapperConfig}
+import uk.gov.hmrc.singlecustomeraccountwrapperdata.config.*
 import uk.gov.hmrc.singlecustomeraccountwrapperdata.controllers.actions.AuthAction
 import uk.gov.hmrc.singlecustomeraccountwrapperdata.models.WrapperDataResponse
-import uk.gov.hmrc.singlecustomeraccountwrapperdata.models.auth.AuthenticatedRequest
 
 import javax.inject.{Inject, Singleton}
 
 @Singleton()
 class ScaWrapperController @Inject() (
   cc: ControllerComponents,
-  appConfig: AppConfig,
-  wrapperConfig: WrapperConfig,
-  urBannersConfig: UrBannersConfig,
-  webchatConfig: WebchatConfig,
+  val appConfig: AppConfig,
+  val wrapperConfig: WrapperConfig,
+  val urBannersConfig: UrBannersConfig,
+  val webchatConfig: WebchatConfig,
```

Comment:
val are not needed as those parameters do not need to be promoted as class fields